Tribler/Core/Modules/restapi/my_channel_endpoint.py

```diff
@@ -3,22 +3,40 @@
 from Tribler.Core.simpledefs import NTFY_CHANNELCAST
 
 
-class MyChannelEndpoint(resource.Resource):
+class MyChannelBaseEndpoint(resource.Resource):
     """
-    This endpoint is reponsible for handing all requests regarding your channel such as getting and updating
-    torrents, playlists and rss-feeds.
+    Base class for all endpoints related to fetching information about my channel.
     """
 
     def __init__(self, session):
         resource.Resource.__init__(self)
         self.session = session
+        self.channel_db_handler = self.session.open_dbhandler(NTFY_CHANNELCAST)
+
+    @staticmethod
+    def return_404(request):
+        """
+        Returns a 404 response code if your channel has not been created.
+        """
+        request.setResponseCode(404)
+        request.finish()
+        return server.NOT_DONE_YET
```

Comment:
I found the code on the Twisted mailing list. I refactored it to make it more understandable and also, a message is returned to the user that the channel has not been created (yet).